lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr_context.hpp

```diff
@@ -93,4 +92,3 @@ status_code_e eps_bearer_deactivate_t3495_handler(zloop_t* loop, int timer_id,
 #ifdef __cplusplus
```

Comment:
Can we take out these functions out of extern C

Reply:
This will be done in a separate PR that I am working on